client/transactions/index.tsx

```diff
@@ -74,7 +77,14 @@ export const TransactionsPage = (): JSX.Element => {
 						},
 						{
 							name: 'uncaptured-page',
-							title: __( 'Uncaptured', 'woocommerce-payments' ),
+							title: sprintf(
+								/* translators: %1: number of uncaptured authorizations */
+								__(
+									'Uncaptured (%1$s)',
+									'woocommerce-payments'
+								),
+								authorizationsSummary.count ?? '...'
```

Comment:
I like this way of simulating a loading 😄 `...`

Reply:
Me too, props should go to @anu-rock in this case 😊

Reply:
Hehe 😃